packages/datadog-plugin-grpc/src/util.js

```diff
@@ -49,7 +48,13 @@ module.exports = {
     }
 
     if (config[filter] instanceof Array) {
-      return element => pick(element, config[filter])
+      return element => {
```

Comment:
This is repeated in multiple places. We should probably have our own `pick` implementation instead of repeating the logic.